tests/test_smoke_tests.py

```diff
@@ -27,12 +27,6 @@ def test_that_server_status_page_loads(self, mozwebqa):
         assert csstat.is_latest_raw_stats_present, 'Raw stats not found'
 
     @pytest.mark.nondestructive
-    def test_that_simple_querystring_doesnt_return_500(self, mozwebqa):
```

Comment:
I never liked this test, I'm glad to see it launched into the sun to burn a quick death.